src/profiler.h

```diff
@@ -47,6 +47,26 @@ enum State {
     TERMINATED
 };
 
+// Keep this in sync with JfrSync
+enum EventMask {
+    EM_CPU          = 1,
+    EM_ALLOC        = 2,
+    EM_LOCK         = 4,
+    EM_WALL         = 8,
+    EM_NATIVEMEM    = 16,
+    EM_METHOD_TRACE = 32,
+    EM_MAX
+};
+constexpr u8 EVENT_MASK_SIZE = __builtin_ctz(EM_MAX - 1) + 1;
+
+static int make_event_mask(const Arguments& args) {
+    return (args._event     != NULL ? (args._latency >= 0 ? EM_METHOD_TRACE : EM_CPU) : 0) |
```

Comment:
Let's leave `event` for ExecutionSamples only.
Separate tracing engine will be only enabled with `trace` option.

Reply:
OK, I hope this will be changed in the follow-up PR for running CPU + tracing engines together.